src/ucp/api/ucp.h

```diff
@@ -704,22 +710,25 @@ ucs_status_t ucp_worker_wait(ucp_worker_h worker);
  * The worker must be armed before waiting on an event (must be re-armed after
  * it has been signaled for re-use) with @ref ucp_worker_arm.
  * The events triggering a signal of the file descriptor from
- * @ref ucp_worker_get_efd , or the return of a call to @ref ucp_worker_wait -
- * depend on the interfaces used by the worker and are defined in the transport
- * layer, and typically represent a request completion or newly available
- * resources. It can also be triggered by calling @ref ucp_worker_signal .
+ * @ref ucp_worker_get_efd are depend on the interfaces used by the worker and
+ * defined in the transport layer, and typically represent a request completion
+ * or newly available resources. It can also be triggered
+ * by calling @ref ucp_worker_signal .
  *
  * @code
  * status = @ref ucp_worker_get_efd (worker, &fd);
  * while (1) {
  *     @ref ucp_worker_arm (worker);
- *     @ref ucp_worker_wait (worker);
+ *     @ref epoll_wait(fd, events, max_events, timeout);
```

Comment:
we don't epoll_wait() on the returned fd, rather need to create a new epoll set which contains it.
maybe would be simpler to demonstrate usage with poll()